managed/Makefile

```diff
@@ -36,8 +36,13 @@ release:                        ## Build pmm-managed release binaries.
 	env CGO_ENABLED=0 go build -v $(PMM_LD_FLAGS) -o $(PMM_RELEASE_PATH)/ ./cmd/...
 	$(PMM_RELEASE_PATH)/pmm-managed --version
 
+ARCH=$(shell uname -m)
 release-dev:
-	go build -race -gcflags="all=-N -l" -v $(PMM_LD_FLAGS) -o $(PMM_RELEASE_PATH)/ ./cmd/...
+	@if [ $(ARCH) = "aarch64" ]; then\
+	    go build -gcflags="all=-N -l" -v $(PMM_LD_FLAGS) -o $(PMM_RELEASE_PATH)/ ./cmd/...
```

Comment:
to ARM do not run with -race


`-race` causes crash building pmm-managed binaries


```
# github.com/percona/pmm/managed/cmd/pmm-managed
reflect.methodValueCall: nosplit stack over 792 byte limit
reflect.methodValueCall<0>
    grows 448 bytes, calls reflect.moveMakeFuncArgPtrs<1>
        grows 288 bytes, calls internal/abi.(*IntArgRegBitmap).Get<1>
            grows 48 bytes, calls runtime.racefuncenter<1>
                grows 0 bytes, calls racefuncenter<121>
                    grows 16 bytes, calls racecall<121>
                        grows 0 bytes, calls indirect
                            grows 0 bytes, calls runtime.morestack<0>
                            8 bytes over limit
```